highfidelity/hifi
# Case 20132, Case 16928: First class Image Entities

https://highfidelity.manuscript.com/f/cases/20132/Create-real-Image-Entities-with-all-the-properties-that-Image-Overlays-have
https://highfidelity.manuscript.com/f/cases/16928/Image-entities-render-edges-in-white-when-image-is-transparent

Test plan:
- Run the content/entity/image tests from this branch: https://github.com/highfidelity/hifi_tests/tree/03d635c22c968bca8ecc73eadcdc2260dd483987/tests/content/entity/image
- Create an Image Entity with a transparent image.  Verify that it is transparent and doesn't have a white border.
